pkg/graphql/request_test.go

```diff
@@ -31,3 +34,53 @@ func TestUnmarshalRequest(t *testing.T) {
 		assert.Equal(t, "query Hello { hello }", request.Query)
 	})
 }
+
+func TestRequest_ValidateForSchema(t *testing.T) {
```

Comment:
I think we need detailed test for each validation error occurring during `ValidateForSchema` instead of saying error count is greater than 0.

Reply:
those different cases are already covered by tests of the library core - it's bad practice to cover cases multiple times in different tests